mix.exs

```diff
@@ -37,8 +37,9 @@ defmodule Guardian.DB.Mixfile do
   defp deps do
     [
       {:guardian, "~> 1.0 or ~> 2.0"},
+      {:plug, "~> 1.8"},
```

Comment:
Why do we need plug there?

Reply:
Good catch. For some reason, while testing I thought guardian_db needed plug. It was a separate matter. Rebased and removed.